collections/_development/code-reviews.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: default
+title:  "Code Reviews"
+area: development
+---
+
+# Code Reviews
+
+Code reviews are an essential part of the development process and must be carried out on all code regardless of where the change originates. The process should also be carried out with the appropriate rigour and attention. Some things to keep in mind during the code review process:
+
+* Give reviews your full attention
+* Limit the amount of time you spend reviewing in any one sitting
+* If necessary check the code out and test it locally
+* Give constructive feedback
+* Consider including non-developers in the review process, they may pick up different issues
+* Do not merge code when there are unresolved comments or requests for change
```

Comment:
I think that it can be mentioned that it is the exception that a developer is merging the code back into master. The QA responsible for testing should be responsible for merging the branch back into master after the piece of work has been PO reviewed.

Reply:
Good call, will add in info around QA and PO involvement in code reviews and the merge process